xml-viewer.html

```diff
@@ -370,14 +370,34 @@
         cssPrefix: this.nodeName.toLowerCase() + ' style-scope '
       });
     }
+    /**
+     * Evaluates if worker can be created on the given URL
+     *
+     * If the worker's hostname is different from the window's hostname it creates a blob URL
+     * to prevent cross domain restrictions
+     *
+     * @param {String} default worker URL
+     * @return {String} URL
+     */
+    generateWorkerUrl(plainUrl) {
+      const workerHostName = new URL(plainUrl).hostname;
+      const windowHostName = window.location.hostname;
+
+      if (workerHostName === windowHostName) return plainUrl;
```

Comment:
This seems to not to solve the problem. It will work on the same domain, cross-domain script will fail anyway because `importScript()` follows the same rules.

Reply:
Hi @jarrodek, thanks for your quick reply! `importScript()` function allows to load external script files or libraries into a worker, so this is a way to hack the browser into importing scripts from another domain. This should fix the CORS issue.

Reply:
But the blob url has `importScript()` with the script location on another domain. I can accept this PR as it is but it won't solve your problem. It will have the same issue.